src/lib/picker.ts

```diff
@@ -687,11 +687,6 @@ export interface PickerOptions {
    * Sets the resolution of recorded video. One of "320x240", "640x480" or "1280x720". Default is `"640x480"`.
    */
   videoResolution?: string;
-  /**
-   * Use Sentry Breadcrumbs mechanism to log information about occured errors.
-   * It can override global objects like console, error etc. Defaults to `true`.
-   */
-  useSentryBreadcrumbs?: boolean;
```

Comment:
I'm unsure how this is used.

This module also needs to be formatted with Prettier, but I minimized the diff for readability.